portality/templates-v2/management/includes/_fixed_form_header.html

```diff
@@ -0,0 +1,24 @@
+<div class="row">
+    <div class="col-md-8 above_the_fold">
+        <h1 class="above_the_fold--title">{% if obj.es_type == constants.LOCK_JOURNAL %}Journal{% elif obj.application_type == constants.APPLICATION_TYPE_UPDATE_REQUEST %}Update Request{% else %}Application{% endif %}: <span id="above_the_fold__title">{{ obj.bibjson().title }}</span> {% set value_elem_id="above_the_fold__title" %}{% include 'includes/_click_to_copy.html' with context%}</h1>
```

Comment:
Technically you can't use `LOCK_JOURNAL` here because it's related to the Lock feature, and may not always be the ES Type for the journal.  A more technically correct comparison would be `Journal.__type__` if this is available in context (perhaps not) and if not what we should do is add `constants.ES_TYPE_JOURNAL = "journal"` and update both `Journal.__type__` and this bit of code to use it.